src/main/java/car/CarRacingGame.java

```diff
@@ -0,0 +1,34 @@
+package car;
+
+import car.model.UserInput;
+import car.view.InputView;
+import car.view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarRacingGame {
+
+    private List<RacingCar> racingCars = new ArrayList<>();
```

Comment:
```suggestion
public class RacingCars {

    private List<RacingCar> racingCars = new ArrayList<>();
```

`List<RacingCar>` 를 멤버로 가진 클래스는 RacingCar 의 일급 컬렉션이 될 수 있도록 설계부탁드립니다.